Server/Terrain.py

```diff
@@ -0,0 +1,91 @@
+import random as ra
+import math as mat
+import bresenham as br
+
+
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 600
+
+BASE_LEVEL = 120
+MIN_LEVEL = 50
+
+MAX_ABS_ANGLE = 15
+NO_ANGLE_BASE_P = 0.45
+
+
+class Terrain:
+    def __init__(self):
+        self.__screen_width = SCREEN_WIDTH
+        self.__screen_height = SCREEN_HEIGHT
+
+        self.__base_level = BASE_LEVEL
+        self.__min_level = MIN_LEVEL
+
+        self.__max_abs_slope = MIN_LEVEL
+        self.__no_slope_base_p = NO_ANGLE_BASE_P
+
+        ra.seed()  # initializing the pseudo-random generator
+        self.__points = [0, self.__screen_width - 1]
+        self.__angles = []
+        self.__levels = [self.__base_level, ]
+
+        self.regenerate_terrain()  # initial terrain generation
+
+    def regenerate_terrain(self):
+        # generating the points for the piecewise linear terrain
+        for useless in range(int(self.__screen_width/100)):  # how many middle points shall be generated
```

Comment:
Khm...useless?

Reply:
I don't need the index variable... thus the name... 
Some suggest using _ but it is said it may mess up the underscore's original functionality - though, risk accepted for now.

Reply:
I think a simple underscore is perfect for "don't care". If there is nothing after it, then it's obviously not a protected variable.